lib/gym/build_command_generator.rb

```diff
@@ -76,8 +76,14 @@ def build_path
       end
 
       def archive_path
+        config = Gym.config
+        unless @archive_path
+          if config[:archive_path]
+            @archive_path = config[:archive_path]
+          end
+        end
```

Comment:
This could be simplified to

```ruby
def archive_path
  @archive_path ||= Gym.config[:archive_path]
  unless @archive_path
    @archive_path ||= [Gym.config[:output_name], Time.now.strftime("%F %H.%M.%S")] # e.g. 2015-08-07 14.49.12
    @archive_path = File.join(build_path, file_name.join(" ") + ".xcarchive")
  end
  return @archive_path
end
```